### PR TITLE
[IA-3209] Enable Rmd file syncing

### PR DESCRIPTION
**Pull request by gpcarr**
See corresponding UI PR for testing https://github.com/DataBiosphere/terra-ui/pull/2810
---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green (you can re-run automation tests with a comment saying `jenkins retest`
- [ ] Run the automation tests multiple times in parallel to weed out instability if applicable via a comment saying `multi-test`
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: http/src/main/resources/init-resources/gce-init.sh [hunk 1]
```diff
@@ -149,6 +142,13 @@ WORK_DIRECTORY='/mnt/disks/work'
 GSUTIL_CMD='docker run --rm -v /var:/var gcr.io/google-containers/toolbox:20200603-00 gsutil'
 GCLOUD_CMD='docker run --rm -v /var:/var gcr.io/google-containers/toolbox:20200603-00 gcloud'
 
+if [ ! -z "$RSTUDIO_DOCKER_IMAGE" ] ; then
+  export IS_RSTUDIO_RUNTIME="true"
+  export XDG_CONFIG_HOME=${RSTUDIO_USER_HOME}/.config/rstudio
```
Comment (Qi77Qi) [inline http/src/main/resources/init-resources/gce-init.sh]: what XDG?

File: http/src/main/resources/reference.conf [hunk 2]
```diff
@@ -493,7 +493,7 @@ gke {
 image {
   welderGcrUri = "us.gcr.io/broad-dsp-gcr-public/welder-server"
   welderDockerHubUri = "broadinstitute/welder-server"
-  welderHash = "342aff2"
+  welderHash = "2bcce86"
```
Comment (Qi77Qi) [inline http/src/main/resources/reference.conf]: this needs to be updated to the hash once welder PR is merged

File: http/src/main/resources/init-resources/gce-init.sh (hunk 1 above)
Comment (gpcarr) [reply http/src/main/resources/init-resources/gce-init.sh]: oh, meant to remove this. it's for autosave which i'll use a separate PR for

**Review by Qi77Qi — APPROVED** [review]

File: http/src/main/resources/reference.conf (hunk 2 above)
Comment (gpcarr) [reply http/src/main/resources/reference.conf]: correct welder hash was merged in this PR https://github.com/DataBiosphere/leonardo/pull/2613

**Pull request merged**